src/redux/services/resource.ts

```diff
@@ -496,7 +496,7 @@ export function reprocessResources(
  * Establishes refs for all resources in specified resourceMap
  */
 
-export function processParsedResources(
+export function processResources(
```

Comment:
It's a bit hard to keep track of functions which have more than 2-3 params, it would be a good idea to send a single params as an object with each function param as a property(you know what goes where and it's less prone to human error in sending wrong params).
I don't know if this falls under this refactoring

Reply:
yeah - sounds like a general refactoring we should do - i can create an issue